src/main/java/pivotaltracker/ui/pages/account/AccountSettingsPage.java

```diff
@@ -90,7 +83,7 @@ public void setIdAccount(final String accountId) {
     /**
      * Account save button.
      */
-    private void clickCreateAccountBtn() {
+    private void clickSaveAccountBtn() {
         saveBtn.click();
```

Comment:
It is realy neccesary to create a Method if you have a single line into?

Reply:
if because the method is private only of the class the actions that are performed should be separated into different methods within the class if there is a need to do more actions should even be an external method as a utility to be able to reuse.